examples/sound/Sound.cpp

```diff
@@ -30,7 +30,7 @@ void playSound()
     sound.play();
 
     // Loop while the sound is playing
-    while (sound.getStatus() == sf::Sound::Playing)
+    while (sound.getStatus() == sf::Sound::Status::Playing)
```

Comment:
Is this intended? This line and a few other lines are related to Audio module.

Reply:
I think that's left around from an early version of this PR where I was also using scoped enums in the audio module. Good catch I'll remove that change.